README.md

```diff
@@ -327,11 +329,14 @@ protoc \
 ### Runtime Library
 
 Pbandk's runtime library provides a Kotlin layer over the preferred Protobuf library for each platform. The libraries are
-present on JCenter. Using Gradle:
+present on Maven Central. Using Gradle:
 
 ```
 repositories {
-    jcenter()
+    // This repository is only needed if using a SNAPSHOT version of pbandk
+    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
@JeroenMols FYI,  if you ever need to test the latest code on `master`, every commit to `master` should now publish artifacts at this maven URL.